app/views/claims/schools/users/index.html.erb

```diff
@@ -7,26 +7,12 @@
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <%= govuk_button_to(t(".add_user"), new_claims_school_user_path, method: :get) %>
-      <% if @users.any? %>
-        <%= govuk_table do |table| %>
-          <% table.with_head do |head| %>
-            <% head.with_row do |row| %>
-              <% row.with_cell(header: true, text: User.human_attribute_name("full_name")) %>
-              <% row.with_cell(header: true, text: User.human_attribute_name("email")) %>
-            <% end %>
-          <% end %>
-          <% table.with_body do |body| %>
-            <% @users.each do |user| %>
-              <% body.with_row do |row| %>
-                <% row.with_cell(text: govuk_link_to(user.full_name, claims_school_user_path(id: user.id))) %>
-                <% row.with_cell(text: user.email) %>
-              <% end %>
-            <% end %>
-          <% end %>
-        <% end %>
-      <% else %>
-        <p><%= t(".no_users", school_name: @school.name) %></p>
-      <% end %>
+
+      <%= render partial: "shared/users_list", locals: {
+        users: @users,
+        school_or_org: @school,
+        user_show_path: "claims_school_user_path",
```

Comment:
Also, it might be more consistent to add a helper to `RouteHelpers` that handles the different paths based on different services. `organisations_path` already exists, we can add `organisation_path(organisation)`. Furthermore, we can just use that url helper in the partial itself.